infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/KubernetesEnvironmentProvisioner.java

```diff
@@ -91,5 +95,8 @@ public void provision(KubernetesEnvironment k8sEnv, RuntimeIdentity identity)
     uniqueNamesProvisioner.provision(k8sEnv, identity);
     ramLimitProvisioner.provision(k8sEnv, identity);
     securityContextProvisioner.provision(k8sEnv, identity);
+
+    // 4 stage - set Kubernetes pod termination grace period
+    podTerminationGracePeriodProvisioner.provision(k8sEnv, identity);
```

Comment:
I'd say that it should be in 3 stage like restartPolicyRewriter and securityContextProvisioner.

Reply:
+1

Reply:
thanks for the pointer will fix